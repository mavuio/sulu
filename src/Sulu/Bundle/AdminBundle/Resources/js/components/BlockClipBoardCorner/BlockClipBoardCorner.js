// @flow
import React from 'react';
import blockclipboardcornerStyles from './blockclipboardcorner.scss';

import BlockClipBoardButton from '../BlockClipBoardButton';

type Props = {|
    value: Object,
    onPasteBlocks: (items: array<object>) => void,
|};

export default class BlockClipBoardCorner extends React.PureComponent<Props> {




    render() {
        const {value,onPasteBlocks} = this.props;

        return (
            <div className={blockclipboardcornerStyles.blockclipboardcorner}>
                    <BlockClipBoardButton value={value} onPasteBlocks={onPasteBlocks} />
            </div>
        );
    }
}


