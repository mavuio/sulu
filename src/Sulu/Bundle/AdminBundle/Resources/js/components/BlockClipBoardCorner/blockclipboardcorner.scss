@import "../../containers/Application/colors.scss";

.blockclipboardcorner {
    flex: 0 0 auto;

    position: absolute;
    right: 0;
    bottom: 0;
}

.blockclipboardcheckbox {
    right: 3px;
    bottom: 3px;
    background: $silver;
}
