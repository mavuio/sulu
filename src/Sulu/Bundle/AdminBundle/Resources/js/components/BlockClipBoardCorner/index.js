// @flow
import BlockClipBoardCorner from './BlockClipBoardCorner';

export default BlockClipBoardCorner;
