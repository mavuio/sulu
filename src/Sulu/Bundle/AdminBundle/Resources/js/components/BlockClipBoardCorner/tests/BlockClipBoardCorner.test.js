// @flow
import React from 'react';
import {render} from 'enzyme';
import BlockClipBoardCorner from '../BlockClipBoardCorner';

test('Render a blockclipboardcorner', () => {
    expect(render(<BlockClipBoardCorner>Hello world</BlockClipBoardCorner>)).toMatchSnapshot();
});
