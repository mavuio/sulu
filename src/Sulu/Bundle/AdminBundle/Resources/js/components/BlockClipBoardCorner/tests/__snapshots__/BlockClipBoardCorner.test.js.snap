// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a blockclipboardcorner 1`] = `
<div
  class="blockclipboardcorner"
>
  Hello world
</div>
`;
